### PR TITLE
Merge cardano-sl-1.0

### PR DESCRIPTION
**Pull request by neongreen**

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -67,7 +67,7 @@ packages:
   extra-dep: true
 - location:
     git: https://github.com/serokell/time-warp-nt.git
-    commit: 7d97bbfb15f1f57f7d9db2aa09c5afe748b0b10d # master v0.2.0.1
+    commit: 0f58f145425650f5e68b838668f36c8342e4a0a8
```
Comment (gromakovsky) [inline stack.yaml]: I think it'd be good to have `# master` comment.

**Comment by neongreen** [comment]
Okay, thanks. I thought this branch already had the workaround 😒

**Comment by gromakovsky** [comment]
Apparently even that workaround doesn't help in 100% case, so you still may need to restart macos build :/

**Comment by gromakovsky** [comment]
> I thought this branch already had the workaround 
There were many workarounds, last one was merged today :)

**Pull request merged**